src/main/java/com/example/demo/entities/Category.java

```diff
@@ -0,0 +1,6 @@
+package com.example.demo.entities;
+
+public enum Category
+{
+    sedan, coupe, hatchback, convertible, sport, suv
```

Comment:
Enum constants should be capitalized. Can create a variable inside the enum and set it to the lowercase value if you need it.